pkg/apis/pipeline/v1alpha1/pipeline_conversion.go

```diff
@@ -0,0 +1,49 @@
+/*
+Copyright 2020 The Tekton Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/tektoncd/pipeline/pkg/apis/pipeline/v1alpha2"
+	"knative.dev/pkg/apis"
+)
+
+var _ apis.Convertible = (*Pipeline)(nil)
+
+// ConvertUp implements api.Convertible
+func (source *Pipeline) ConvertUp(ctx context.Context, obj apis.Convertible) error {
+	switch sink := obj.(type) {
+	case *v1alpha2.Pipeline:
+		sink.ObjectMeta = source.ObjectMeta
+		return nil // source.Spec.ConvertUp(ctx, &sink.Spec)
```

Comment:
This is a placeholder :angel: I needed to define at least 2 Convertible types to some tests :stuck_out_tongue_closed_eyes: